projects/examples/src/components/form-input/number-with-unit-form-input-unitless.example.component.html

```diff
@@ -0,0 +1,42 @@
+<form [formGroup]="formGroup" class="clr-form-horizontal">
```

Comment:
Can we add some text to the example giving an overview of the feature we're exemplifying?